packages/waku/src/lib/plugins/vite-plugin-rsc-hmr.ts

```diff
@@ -126,10 +126,11 @@ export function rscHmrPlugin(): Plugin {
     handleHotUpdate({ file }) {
       const moduleLoading = (globalThis as any).__webpack_module_loading__;
       const moduleCache = (globalThis as any).__webpack_module_cache__;
+      if (file.startsWith(viteServer.config.root)) {
+        file = file.slice(viteServer.config.root.length);
+      }
       const id = filePathToFileURL(file);
-      if (moduleLoading.has(id) && moduleCache.has(id)) {
-        moduleLoading.delete(id);
```

Comment:
Why don't we delete anymore?

Reply:
I thought checking `moduleCache.has(id)` is unnecessary. What do you think?

Reply:
I do not remember exactly where the condition might happen, but I think having it might be safer! So a new module gets loaded later anyway.

Reply:
Hm? I think removing it might be safer.